contracts/Strategy.sol

```diff
@@ -311,12 +313,14 @@ contract Strategy is BaseStrategy {
     }
 
     function _checkCooldown() internal view returns (CooldownStatus) {
```

Comment:
here you are not taking into account the case that the unstake window has passed. 

it will return CooldownStatus.None

maybe worth it to add to tests + explicitly add it to the case the cooldown has not been initiated

just to avoid getting locked into the strat if something doesn't work as expected

Reply:
If it's passed, it's equivalent to none. I've tested the case where it's passed and needs reinit (see: https://github.com/fp-crypto/stkaave-cooldown/blob/main/tests/test_operation.py#L102-L123)